contracts/PensionFundRelease.sol

```diff
@@ -7,10 +7,16 @@ contract PensionFundRelease {
     address[] public validators;
     address public worker;
     uint8 public firstPaymentPercent;
-    uint public firstPaymentTime;
+    uint8 public reccurentPaymentPercent;    
+    uint public paymentTime;
+    uint public lastPaymentTime;
     uint public reccurentPaymentInterval;
-    uint8 public reccurentPaymentPercent;
+    uint public totalPayment;
```

Comment:
I haven't noticed where you are using these variables (totalPayment, totalReleased, lastPaymentTime)